doc/languages.rst

```diff
@@ -169,9 +169,15 @@ to view the docstring of the syntax checker.  Likewise, you may use
          A list of include directories.  Relative paths are relative to the file
          being checked.
 
-      .. option:: flycheck-cppcheck-language-standard
+      .. option:: flycheck-cppcheck-standards
 
-         The C or C++ language standard to use via ``--std=``.
+         The C,  C++ and/or  POSIX standards  to use  via one  or more
+         ``--std=``.
+
+      .. option:: flycheck-cppcheck-suppressions
+
+         The  cppcheck  suppressions  list  to use  via  one  or  more
+         ``--suppress=``.
```

Comment:
@fredmorcos Oh, indeed?!  My `M-q` doesn't insert double-spaces after every word 😕 What setting did you change to achieve this effect?  I've never seen this behaviour before.

But in any case, yes, please change that.  While we use double-spacing at sentence end in line with Emacs' default settings, we definitely don't put two spaces after every single word.